Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -168,23 +168,6 @@ void VideoConfig::GameIniLoad()
 		{
 			iEFBScale = tmp;
 		}
-		else // Round down to multiple of native IR
```

Comment:
I think it should be fine to read iEFBScale directly:
CHECK_SETTING("Video_Settings", "EFBScale", iEFBScale );
Maybe you also need to touch all affected game inis :/